code/unit_tests/map_tests.dm

```diff
@@ -633,6 +634,9 @@
 	if(!isStationLevel(source_turf.z))
 		return TRUE
 
+	if(locate(/obj/abstract/landmark/skip_test) in source_turf)
```

Comment:
Is this the only test this should be checked?

Reply:
For now yes. But it's a work in progress tool for convenience.